spring/spring-web-reactive/src/main/java/io/quarkus/ts/spring/web/reactive/bootstrap/persistence/model/Book.java

```diff
@@ -80,11 +80,9 @@ public boolean equals(Object obj) {
         if (id != other.id)
             return false;
         if (title == null) {
-            if (other.title != null)
-                return false;
-        } else if (!title.equals(other.title))
-            return false;
-        return true;
+            return other.title == null;
+        } else
```

Comment:
FYI in QE FW there is a rule that you need to use curly brackets. This was part of the original code as well and validation succeeded so np.